src/patternfly/utilities/variables.scss

```diff
@@ -202,6 +206,9 @@ $pf-global--LineHeight--md: 1.5 !default;
 // List
 $global-ListStyle: disc outside !default;
 
+// Arrow size (used for tooltips, popovers, etc.)
```

Comment:
I like this, we should create a story to use it on drop menus too.
I'd change px units to `pf-size-prem(25px)`

Reply:
I see this as a different thing than the arrows that display in dropdowns. Wouldn't the arrows we use for dropdowns be sized relative to font size?

Reply:
@andresgalante  - based on @jgiardino's comment, is your direction to still use pf-size-prem(25px) ?